src/lib/oneElLambda.cpp

```diff
@@ -21,14 +21,14 @@ void oneElLambda::interp(const oneElLambda &previousEl,
                          const oneElLambda &nextEl) {
   // normal linear interpolation possible (the two lambda are positive and the
   // next value is above the previous one
-  if (previousEl.lamb > 0 && nextEl.lamb > previousEl.lamb) {
+  if (previousEl.lamb >= 0 && nextEl.lamb > previousEl.lamb) {
```

Comment:
We are overloading the semantics here. the only problematic case with the interpolation is the 2 lambdas being equal. What are we trying to achieve here with checking lambda ordering of the 2 arguments?

Reply:
the comment says : 
- the two lambdas are positive : I agree that this is a specifics of the oneElLambda semantics. And we could enforce it when creating a oneElLambda
- the next val is above the previous val : this is by no means necessary for interpolation to work, so there is no reason to force this condition here, where it is not understandable.